homeassistant/components/binary_sensor/translations/en.json

```diff
@@ -159,8 +163,8 @@
             "on": "Plugged in"
         },
         "presence": {
-            "off": "Away",
-            "on": "Home"
+            "off": "[%key:common::state::not_home%]",
+            "on": "[%key:common::state::home%]"
```

Comment:
This is unrelated and should be changed in a seperate PR

Reply:
I do not have any idea, why I've changed this 🤔 🤷‍♂️

Reply:
ahh ... `python -m script.translations develop` does this change ... but as I understand, this should not be done 🤔

Reply:
Probably need to rebase your branch

Reply:
rebase does not fix it ... however, I will re-fix/remove it again ...